crates/core/tedge_actors/src/servers/mod.rs

```diff
@@ -144,20 +142,12 @@ impl<Request, Response> AsRef<Request> for RequestEnvelope<Request, Response> {
     }
 }
 
-/// A request sender to some [Server]
-pub type DynRequestSender<Request, Response> = DynSender<RequestEnvelope<Request, Response>>;
-
 /// A connector to a [Server] expecting Request and returning Response.
 pub trait Service<Request: Message, Response: Message>:
     MessageSink<RequestEnvelope<Request, Response>>
 {
     /// Connect a request message box to the server box under construction
     fn add_requester(&mut self, response_sender: DynSender<Response>) -> DynSender<Request>;
```

Comment:
```suggestion
    fn connect_client(&mut self, response_sender: DynSender<Response>) -> DynSender<Request>;
```
Optional, now that the other `add_client` method is gone.